### DISCUSSION TITLE
Where is the tutorial of this community?

### DISCUSSION THREAD

**Discussion by shijiezhou1** [community/community#39042] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Where is the tutorial of this community?

**Comment by alimttth** [comment]
Hello, the best tutorials are on YouTube, go there
Have a good fast

**Comment by DSDmark** [comment]
Hi @shijiezhou1,

if you want to read about **github-community** then you could read this:- https://github.com/community

you can also this check this out:-
(https://github.com/orgs/community/discussions/28776)
